src/atisha/Assignment78.java

```diff
@@ -0,0 +1,40 @@
+//*     1             1
+//**    1 2           2 3
+//***   1 2 3         4 5 6
+//****  1 2 3 4       7 8 9 10
+//      1 2 3 4 5
+//      1 2 3 4 5 6
+package atisha;
+
+public class Assignment78 {
+	public static void main(String[] args) {
+		System.out.println("pattern->1");
+		for (int i = 0; i <= 4; i++) {
```

Comment:
Make code generic so if you want to print for 10 or 2 rows then also it is possible